client/jetpack-connect/jetpack-connection.jsx

```diff
@@ -73,6 +83,19 @@ const jetpackConnection = ( WrappedComponent ) => {
 						{ translate( 'Install Jetpack manually' ) }
 					</LoggedOutFormLinkItem>
 					<HelpButton />
+					{ this.canGoBack() && (
+						<div className="jetpack-connect__navigation">
+							<Button
+								compact
+								borderless
+								className="jetpack-connect__back-button"
+								onClick={ this.goBack }
+							>
+								<Gridicon icon="arrow-left" size={ 18 } />
```

Comment:
Should use wordpress icon not gridicon here: `<Button icon={arrowLeft} />`

Reply:
I guess Osk chose this because it's already used elsewhere in JPC that way. If we replace those with `@wordpress` icons, we might want to do it consistently